conconartist/face-it
# Sync backend

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 

Successfully synced database via MySQL server on local machine and refactored the frontend to display new changes.

### Problems Encountered & Resulting Solutions  

A few bugs occurred due to the new data format, so some time was spent refactoring.


### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

App.js in the Components folder, including new components being added, all under the Category folder.

### How should this be tested?
> What steps need to be taken to confirm functionality? 
Go to the faceit-sql-server project in your terminal directory (if it's not already installed, clone the repo and install necessary packages). Start the server with `nodemon server.js`.
In a new window/ tab, go to the front end project directory and run `npm start`.  Open `http://localhost:3000` to view the site in your browser and interact with the site as a user.
